portality/migrate/1779_fix_misrelease/README.md

```diff
@@ -0,0 +1,7 @@
+*S.E. 2021-11-01*
+
+Save (and re-generate index) for Applications created during the window where we mislabelled them by releasing code early.
+
+Run
+
+    python portality/upgrade.py -u portality/migrate/1779_fix_misrelease/migrate.json
```

Comment:
Our new protocol is to label the migration scripts directories with the date they were created